### ISSUE TITLE
Unable to send the key: connect ECONNREFUSED

### ISSUE THREAD

**Issue by locnho** [kkapitan/homebridge-homesung#11]
Hi,

I got this all working. But once I power off the TV and start over again, I get "Unable to send the key: connect ECONNREFUSED 192.168.1.11:8000" on the Homebridge console. Anyone see this and how to fix it?

**Comment by lasry1** [comment]
It happens to me too, it looks like the TV renew lease of the key (maybe a protection of the TV?) i haven't been able to sort it

**Comment by locnho** [comment]
Hi,

Can’t we just change to re-do the connection again when this failed. Try for three time and give up.

-Loc

**Comment by kaosmagix** [comment]
This also happens with me. I however seems related to a time-out as well as it doesn't always happen. I sometimes can turn on/off the TV multiple times before this happens

**Comment by locnho** [comment]
I can't say for sure but it seems like an issue when you power off from the remote instead from the HomeKit.

**Comment by locnho** [comment]
I am not sure it is related. But while working on my project - https://github.com/locnho/homebridge-vsx, I notice that I get ECONNREFUSED too. But it also complain about emitter limit. After set the emitter limit to 100, it works just fine with my Pioneer SC-1223K receiver.

**Comment by kaosmagix** [comment]
@locnho Are you saying you can't use the remote to power off anymore? Tat would be a shame, but let me test some more.

**Comment by locnho** [comment]
Yes... Once I get the ECONNREFUSED, the Homekit App control will NO longer work until I re-start the homebridge application. The homesung plugin keeps getting ECONNREFUSED error as it won't be able to communicate with the Samsung H6300 TV.
